frontend/src/views/PredatorPrey/PredatorPrey.vue

```diff
@@ -395,7 +407,7 @@ export default {
         console.log("Preset deleted");
       } catch (error) {
         const failureAlertPayload = {
-          message: "Unable to delete preset, failed repsonse from server",
+          message: "Unable to delete preset, failed repsonse from server. Please try again at another time",
```

Comment:
```suggestion
          message: "errorMsg(“delete preset”)",
```

you have the majority of this error message repeated a load of times, you could create function with some variation like passing the error type above to reduce the repeated code